config/locales/server.en.yml

```diff
@@ -2811,848 +2811,3368 @@ en:
       invalid_uncategorized_category_setting: 'The "Uncategorized" category cannot be selected if ''allow uncategorized topics'' is not enabled.'
       invalid_search_ranking_weights: "Value is invalid for search_ranking_weights site setting. Example: '{0.1,0.2,0.3,1.0}'. Note that maximum value for each weight is 1.0."
 
+    # Run `RAILS_ENV=development bundle exec rake site_settings:add_keyword_translation_keys` to update this section.
+
     keywords:
-      # Run `RAILS_ENV=development bundle exec rake site_settings:add_keyword_translation_keys` to update this section.
       # BEGIN KEYWORDS
+
```

Comment:
Is there a way to discourage Crowdin from using a comment for the context entry? I'm hoping adding the blank line will do it, but I'm not sure if that will work.